sagol/models/svr.py

```diff
@@ -1,9 +1,44 @@
 import numpy as np
 
 from sklearn.svm import SVR
-
+from sklearn.model_selection import GridSearchCV
+from sklearn.model_selection import train_test_split
 
 def train_svr(x_train: np.ndarray, y_train: np.ndarray, **kwargs) -> SVR:
+    print("train size is now" ,len(x_train))
+    print("first data is ", x_train[0])
+    print("first label is ", y_train[0])
     mdl = SVR(**kwargs)
     mdl.fit(x_train, y_train)
     return mdl
+
+def run_svr_grid_search(data):
+    # TODO: get parameters for the grid search, the ones here are copied from:
+    #https://scikit-learn.org/stable/auto_examples/plot_kernel_ridge_regression.html#sphx-glr-auto-examples-plot-kernel-ridge-regression-py
+    #TODO: understand how the gridSearch uses test set (currently it does not, I guess it does k fold cross validation)
```

Comment:
It does not, right now the implementation "throws away" the test data.

The `cv` param is the `k` of the `k-fold` cross validation.
See: https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GridSearchCV.html